nntrainer/include/nntrainer_log.h

```diff
@@ -0,0 +1,68 @@
+/**
+* Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @file nntrainer_log.h
+ * @date 06 April 2020
+ * @brief NNTrainer Logger.
+ *        Log Util for NNTrainer
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#ifndef __NNTRAINER_LOG_H__
+#define __NNTRAINER_LOG_H__
+
+#define TAG_NAME "nntrainer"
+
+#if defined(__TIZEN__)
+#include <dlog.h>
+
+#define nn_logi(...) dlog_print(DLOG_INFO, TAG_NAME, __VA_ARGS__)
+
+#define nn_logw(...) dlog_print(DLOG_WARN, TAG_NAME, __VA_ARGS__)
+
+#define nn_loge(...) dlog_print(DLOG_ERROR, TAG_NAME, __VA_ARGS__)
+
+#define nn_logd(...) dlog_print(DLOG_DEBUG, TAG_NAME, __VA_ARGS__)
+n
+#elif defined(__ANDROID__)
+#include <android/log.h>
+
+#define ml_logi(...) __android_log_print(ANDROID_LOG_INFO, TAG_NAME, __VA_ARGS__)
+
+#define ml_logw(...) __android_log_print(ANDROID_LOG_WARN, TAG_NAME, __VA_ARGS__)
+
+#define ml_loge(...) __android_log_print(ANDROID_LOG_ERROR, TAG_NAME, __VA_ARGS__)
+
+#define ml_logd(...) __android_log_print(ANDROID_LOG_DEBUG, TAG_NAME, __VA_ARGS__)
+
+#else /* Linux distro */
+#include <nntrainer_logger.h>
+
+#define ml_logi(...) __nntrainer_log_print(NNTRAINER_LOG_INFO, __VA_ARGS__, NULL)
+
+#define ml_logw(...) __nntrainer_log_print(NNTRAINER_LOG_WARN, __VA_ARGS__, NULL)
+
+#define ml_loge(...) __nntrainer_log_print(NNTRAINER_LOG_ERROR, __VA_ARGS__, NULL)
+
+#define ml_logd(...) __nntrainer_log_print(NNTRAINER_LOG_DEBUG, __VA_ARGS__, NULL)
+#endif
+
+#define nn_logi ml_logi
+#define nn_logw ml_logw
+#define nn_loge ml_loge
+#define nn_logd ml_logd
```

Comment:
I think :32~:38 should be ml_* otherwise macro seems to be overriden.

Reply:
fixed.